samples/tab-mail/ts/README.md

```diff
@@ -0,0 +1,93 @@
+---
+page_type: sample
+description: Sample tab-mail app.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- typescript
+extensions:
+ contentType: samples
+ createdDate: "20/06/2023 01:38:27 PM"
+urlFragment: officedev-microsoft-teams-samples-tab-mail-ts
+---
+
+# Tab-Mail
+
+This sample helps to compose mail from perosnal tab.
+
+ ## Included Features
+* Tabs
+
+## Interaction with app
+
+![Tab-Mail](Images/tab-mail.gif)
+
+## Prerequisites
+-  [NodeJS](https://nodejs.org/en/)
+
+-  [M365 developer account](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant) or access to a Teams account with the appropriate permissions to install an app.
+- [Ngrok](https://ngrok.com/download) (For local environment testing) Latest (any other tunneling software can also be used)
+  
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## Setup
+    
+1. Setup NGROK
+  - Run ngrok - point to port 3978
+
+    ```bash
+    ngrok http 3978 --host-header="localhost:3978"
+    ```
+2. Setup for code
+ - Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+-  In a terminal, navigate to `samples/tab-mail/ts`
+
+ - Install modules
+
+    ```bash
+    npm install
+    ```
+ - Run your app
+
+    ```bash
+    npm start
+    ```
+3. Setup Manifest for Teams
+
+- **This step is specific to Teams.**
+    **Edit** the `manifest.json` contained in the `Manifest/` folder to replace with your Manifest-Id *everywhere* you see the place holder string `{{Manifest-Id}}`.
```

Comment:
How to get this {{Manifest-Id}} ? and what is the valid value for it?

Reply:
fixed